src/java/org/apache/cassandra/index/sai/disk/format/Version.java

```diff
@@ -25,22 +34,46 @@
  */
 public class Version
 {
-    private static final Version AA = new Version('a', 'a');
+    // 6.8 formats
+    public static final Version AA = new Version("aa", V1OnDiskFormat.instance);
+    // Stargazer
+    public static final Version BA = new Version("ba", V1OnDiskFormat.instance);
```

Comment:
What's the point of having different versions with the same format?

Reply:
I wanted to have a new version using the new naming layout for the initial Stargazer release. It's not necessary but I thought it would be a good idea because it means we can make on-disk changes if we want without having to worry about it.